templates/bridgeserver2.yaml

```diff
@@ -0,0 +1,834 @@
+          - 412Count
+      Namespace: LogMetrics/412s
+      Period: 3600
+      Statistic: Sum
+      Threshold: 30
+      TreatMissingData: notBreaching
+  AWSCWCpuUtilizationAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AWSSNSTopic
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      EvaluationPeriods: 1
+      MetricName: CPUUtilization
+      Namespace: AWS/EC2
+      Period: 900
+      Statistic: Average
+      Threshold: 80
+  AWSCWEnvironmentHealthAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AWSSNSTopic
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      EvaluationPeriods: 1
+      MetricName: EnvironmentHealth
+      Namespace: AWS/ElasticBeanstalk
+      Period: 900
+      Statistic: Maximum
+      Threshold: 25
+  AWSCWHealthCheckStatusAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AWSSNSTopic
+      ComparisonOperator: LessThanThreshold
+      EvaluationPeriods: 1
+      MetricName: HealthCheckStatus
+      Namespace: AWS/Route53
+      Dimensions:
+        -
+          "Name": "HealthCheckId"
+          "Value": !Ref AWSR53HealthCheck
+      Period: 300
+      Statistic: Average
+      Threshold: 1
+  AWSLBSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: Load Balancer Security Group
+      VpcId: !Ref AwsDefaultVpcId
+      SecurityGroupIngress:
+        - CidrIp: 0.0.0.0/0
+          FromPort: '443'
+          ToPort: '443'
+          IpProtocol: tcp
+        - CidrIp: 0.0.0.0/0
+          FromPort: '80'
+          ToPort: '80'
+          IpProtocol: tcp
+      SecurityGroupEgress:
+        - CidrIp: 0.0.0.0/0
+          FromPort: '80'
+          ToPort: '80'
+          IpProtocol: tcp
+  AWSEC2SecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    DependsOn: AWSLBSecurityGroup
+    Properties:
+      GroupName: !Join
+        - '-'
+        - - !Ref 'AWS::StackName'
+          - AWSEC2SecurityGroup
+      GroupDescription: EC2 Security Group
+      VpcId: !Ref AwsDefaultVpcId
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: '80'
+          ToPort: '80'
+          SourceSecurityGroupId: !Ref AWSLBSecurityGroup
+  # Tests running on travis requires direct access to RDS (should only apply to Dev RDS instance)
+  # https://docs.travis-ci.com/user/ip-addresses/
+  AWSEC2RdsOpenSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    DependsOn: AWSEC2SecurityGroup
+    Properties:
+      GroupDescription: !Join
+        - '-'
+        - - !Ref 'AWS::StackName'
+          - AWSEC2RdsOpenSecurityGroup
+      VpcId: !Ref AwsDefaultVpcId
+      SecurityGroupIngress:
+        - CidrIp: '0.0.0.0/0'
+          FromPort: '3306'
+          ToPort: '3306'
+          IpProtocol: tcp
+  IAMBridgeServerECManagedPolicy:
```

Comment:
The project is BridgeServer2 specifically to distinguish it from BridgePF (which is basically BridgeServer1).

I can open a discussion at the next standup to ask if people would prefer we name the stack something generic (BridgeServer) or something specific to this version of the stack (BridgeServer2). My feeling is that the stack itself is not generic (since it's separate from BridgePF), nor is it future-proof (Bridge Server v3.0 may have different requirements, if it ever exists), so it's not particularly useful nor meaningful to name it something generic or future-proof.

Reply:
I'm just suggesting that either name everything BridgeServer or BridgeServer2.  if you decide on BridgeServer2 then maybe name this resource as `IAMBridgeServer2ECManagedPolicy`?

Reply:
Changed all instances of BridgeServer to BridgeServer2.